modules/nw-routeadvertisements-object.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * networking/route_advertisements/about-route-advertisements.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-bgp-routeadvertisements-object_{context}"]
+= RouteAdvertisements object configuration
+
+You can define an `RouteAdvertisements` object, which is cluster scoped, with the following properties.
+
+The fields for the `RouteAdvertisements` custom resource (CR) are described in the following table:
+
+.`RouteAdvertisements` object
+[cols=".^2,.^2,.^6",options="header"]
+|====
+|Field|Type|Description
+
+|`metadata.name`
+|`string`
+|Specifies the name of the `RouteAdvertisements` object.
+
+|`advertisements`
+|`array`
+|Specifies an array that can contain a list of different types of networks to advertise. Supports only the `podNetwork` and `egressIP` values.
+
+|`frrConfigurationSelector`
+|`object`
+|Optional: Determines which `FRRConfiguration` CR the OVN-Kubernetes driven `FRRConfiguration` CR is based on. When omitted, all `FRRConfiguration` CRs are considered.
+
+|`networkSelector`
+|`object`
+|Optional: Specifies which networks to advertise. When omitted, advertises all networks. To select the default network, match the predefined `k8s.ovn.org/default-network` label.
```

Comment:
The field changed to `networkSelectors`
It is no longer `Optional`, now `Required`
I would explain as `Specifies which networks to advertise among default cluster network and cluster user defined networks`